PWGGA/GammaConv/macros/AddTask_GammaHeavyMeson_CaloMode_pp.C

```diff
@@ -364,6 +364,29 @@ void AddTask_GammaHeavyMeson_CaloMode_pp(
     cuts.AddCutCalo("00010113","2446600040013300000","0163103b00000010"); // INT7
     cuts.AddCutCalo("00062113","2446600040013300000","0163103b00000010"); // PHI7
 
+    // *****************************************************************************************************
+    // 13 TeV pp Run2 - EDC (EMCal + DCal) configurations
+    // *****************************************************************************************************
+
+  } else if (trainConfig == 2020){ // EMCAL+DCAL clusters standard cuts, INT7, NL , std TM, tight timing
+    cuts.AddCutCalo("00010113","41179000a7032230000","01631031000000d0"); // INT7 No NL
```

Comment:
Hi Joshua,
i highly recommend to not use the tight timing cut "a". This cut kills a lot of your triggered yields. 
Rather use +-25 or 30 ns (cutnumber 7 or 8).

Reply:
Hi Nico,
you are absolutely right. Thanks for checking so carefully!! I have changed the time cut to 6, which we had for this analysis before.